zstd_compression_format.md

```diff
@@ -97,6 +97,42 @@ to decode all concatenated frames in their sequential order,
 delivering the final decompressed result as if it was a single content.
 
 
+Skippable Frames
```

Comment:
Skippable Frames are an oddity,
not really the main objective of the specification.

Is there any gain in putting it at the beginning of the document, rather than the end ?